### PR TITLE
Add Mood Tracker plugin

### PR DESCRIPTION
**Pull request by dartungar**
# I am submitting a new Community Plugin

## Repo URL

Link to my plugin: [Mood Tracker](https://github.com/dartungar/obsidian-mood-tracker)

## Release Checklist
- [x] I have tested the plugin on
  - [x]  Windows
  - [ ]  macOS
  - [ ]  Linux
  - [ ]  Android _(if applicable)_
  - [ ]  iOS _(if applicable)_
- [x] My GitHub release contains all required files
  - [x] `main.js`
  - [x] `manifest.json`
  - [x] `styles.css` _(optional)_
- [x] GitHub release name matches the exact version number specified in my manifest.json (_**Note:** Use the exact version number, don't include a prefix `v`_)
- [x] The `id` in my `manifest.json` matches the `id` in the `community-plugins.json` file.
- [x] My README.md describes the plugin's purpose and provides clear usage instructions.
- [x] I have read the tips in https://github.com/obsidianmd/obsidian-releases/blob/master/plugin-review.md and have self-reviewed my plugin to avoid these common pitfalls.
- [x] I have added a license in the LICENSE file.
- [x] My project respects and is compatible with the original license of any code from other plugins that I'm using.
      I have given proper attribution to these other projects in my `README.md`.

### CODE REVIEW

**Comment by joethei** [comment]
["description": "Mood Tracker for Obsidian.md",](https://github.com/dartungar/obsidian-mood-tracker/blob/a585c9dd524f36005284956d2e6798632ffd0345/manifest.json#L6)
We have recently published a guide for plugin descriptions to follow here: https://docs.obsidian.md/Plugins/Releasing/Submission+requirements+for+plugins#Keep+plugin+descriptions+short+and+simple.
This being a plugin for Obsidian is obvious and can be removed, same with this being a mood tracker, the name already makes that clear.

[Copyright (c) [year] [fullname]](https://github.com/dartungar/obsidian-mood-tracker/blob/a585c9dd524f36005284956d2e6798632ffd0345/LICENSE#L3)
Fill this with your own info.

**Comment by dartungar** [comment]
@joethei thanks for the note! I've tried to make a more robust (yet succinct) description:
> Track your moods & emotions easily. Visualize tracked history and browse the past entries.

**Comment by dartungar** [comment]
@marcusolsson just a friendly reminder about this humble pull request 🥺

**Comment by marcusolsson** [comment]
["1.0.0": "0.15.0"](https://github.com/dartungar/obsidian-mood-tracker/blob/2857455d2d3fd5cf5fe67f396a880f1ebec1dfb0/versions.json#L2)
Did you update the versions.json? It says that users need version 0.15.0 of your plugin if they're using Obsidian 1.0.0 or later. And I can only see version 0.1.0 in your repo.

[containerEl.createEl('h2', { text: 'Mood Tracker Settings' });](https://github.com/dartungar/obsidian-mood-tracker/blob/2857455d2d3fd5cf5fe67f396a880f1ebec1dfb0/src/settings/settingsTab.ts#L18)
https://docs.obsidian.md/Plugins/Releasing/Plugin+guidelines#UI+text

**Comment by dartungar** [comment]
@marcusolsson thanks, I've fixed versions.json & removed the heading.

File: community-plugins.json [hunk 1]
```diff
@@ -6885,6 +6885,12 @@
     "name": "Rofi Helper",
     "description": "Adds a leaf ID parameter to the URI protocol for switching between open obsidian tabs with Rofi. A sample Rofi script is included.",
     "author": "digitalsignalperson",
-    "repo": "digitalsignalperson/obsidian-rofi-helper"
+  },
+  {
```
Comment (liamcain) [inline community-plugins.json:6889]:
```suggestion
        "repo": "digitalsignalperson/obsidian-rofi-helper"
  },
  {
```

File: community-plugins.json [hunk 2]
```diff
@@ -6885,6 +6885,13 @@
     "name": "Rofi Helper",
     "description": "Adds a leaf ID parameter to the URI protocol for switching between open obsidian tabs with Rofi. A sample Rofi script is included.",
     "author": "digitalsignalperson",
-    "repo": "digitalsignalperson/obsidian-rofi-helper"
+        "repo": "digitalsignalperson/obsidian-rofi-helper"
```
Comment (liamcain) [inline community-plugins.json:6888]:
```suggestion
    "repo": "digitalsignalperson/obsidian-rofi-helper"
```

**Pull request merged**